### ISSUE TITLE
Departure at KORD

### ISSUE THREAD

**Issue by panther2** [openscope/openscope#320]
When going to KORD, I start with departures like these:
![open](https://cloud.githubusercontent.com/assets/20812775/25056285/2b5d2aee-2168-11e7-939e-f49f264eb9d3.PNG)

I cannot see, to which departures those planes are going, without `caf` as first command.
So in order to to find a reasonable starting sequence I have to `caf` them all before I `taxi` them.
Is this intentional?

**Comment by headphase** [comment]
The departure in all of those examples is the O'Hare One (ORD1) _[which should actually be updated to the O'Hare TWO to match the most recent real life revision, but that's for another time]_.  so this is probably me being pedantic, but what you probably are asking is for the transition (exit waypoint) to be displayed?

I can't speak to intentions of flight strip behavior in the sim (hopefully someone else can chime in), but in the real world, controllers don't have much luxury of deciding launch order as a way of making life easier  :stuck_out_tongue:  

Sure they can issue release times to hold aircraft in certain scenarios but for the most part, and especially at busy airports, the Tower (local) controller will be handing out takeoff clearances as planes reach the end of the runway in sequence, in conveyor-belt-fashion.  So in that sense, it doesn't really matter where each of those planes expects vectors to because in real life the approach controller needs to make the puzzle pieces fit together once they're airborn.

**Comment by n8rzz** [comment]
re: `aircraftStripView`, here are the various phases and appearances of a strip (for an aircraft at ksfo)

<img width="747" alt="screen shot 2017-04-14 at 6 16 15 pm" src="https://cloud.githubusercontent.com/assets/3325985/25058416/b06052e0-213e-11e7-920c-20b66bb7cc05.png">

**Comment by panther2** [comment]
Thanks for the insight, @headphase and the summary, @n8rzz . So this appears to be intentional.
I have been asking, because it had been different at KORD in the past, IIRC.

**Comment by erikquinn** [comment]
> I can't speak to intentions of flight strip behavior in the sim (hopefully someone else can chime in), but in the real world, approach (TRACON) controllers don't have much luxury of deciding launch order as a way of making life easier

True, but that's a big part of what ground should be doing. Frequently, aircraft over different fixes will be assigned different runways, and sorted in a way that yields the best variation (since you can then launch the last aircraft sooner than if you had to wait for spacing between successive aircraft over the same fix). So in effect, you're doing what ground would be doing: trying to _give_ the TRACON aircraft that naturally separate themselves.

> In other words, takeoff (and taxi) priority depends on who calls for clearance first.

Yup. When the line starts to back up more than one or two though, ground (_should_) begin building the departure sequence for the maximum overall efficiency.

**Comment by erikquinn** [comment]
And with #18/#295, the strip format will be changed so you can see the entire route. They will lose their interactive coloring, and instead be static flight strips formatted to look like the real ones.

**Comment by panther2** [comment]
So now with 5.0.0 all departures go to `ord1.noony` ... 
I need to issue the `say route` command to even get an idea about where they might be going  - and there is no departing plane going elsewhere than `noony`
